package.json

```diff
@@ -25,7 +25,10 @@
     ],
     "exclude": [
       "tests/**/*Tests.ts",
-      "src/data_vis.ts"
+      "src/data_vis.ts",
+      "src/Data.ts",
```

Comment:
Explicitly exclude these troublesome to test files, which for some reason started bothering the codecov reports (we never had tests for these as far as I can tell).